internal/pkg/agent/application/monitoring/process.go

```diff
@@ -22,12 +22,13 @@ import (
 )
 
 const (
-	componentIDKey    = "componentID"
-	metricsPathKey    = "metricsPath"
-	timeout           = 10 * time.Second
-	apmPrefix         = "apm-server"
-	apmTypePrefix     = "apm"
-	fleetServerPrefix = "fleet-server"
+	componentIDKey         = "componentID"
+	metricsPathKey         = "metricsPath"
+	timeout                = 10 * time.Second
+	apmPrefix              = "apm-server"
+	apmTypePrefix          = "apm"
+	fleetServerPrefix      = "fleet-server"
+	profilingServicePrefix = "pf-elastic-"
```

Comment:
I think this one needs to be literally 
```suggestion
	profilingServicePrefix = "pf-elastic-symbolizer"
```

Reply:
nvm, resolved 😄 apologies...